src/Aquifer.Jobs/Subscribers/NotificationSubscriber.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Data.Common;
+using System.Web;
+using Aquifer.Common.Jobs;
+using Aquifer.Common.Jobs.Messages;
+using Aquifer.Common.Services;
+using Aquifer.Data;
+using Aquifer.Jobs.Configuration;
+using Azure.Storage.Queues.Models;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Aquifer.Jobs.Subscribers;
+
+public sealed class NotificationSubscriber(
+    IOptions<ConfigurationOptions> _configurationOptions,
+    AquiferDbContext _dbContext,
+    IEmailService _emailService,
+    ILogger<NotificationSubscriber> _logger)
+{
+    [Function(nameof(SendProjectStartedNotification))]
+    public async Task SendProjectStartedNotification(
+        [QueueTrigger(Queues.SendProjectStartedNotification)] QueueMessage queueMessage,
+        CancellationToken ct)
+    {
+        var message = queueMessage.Deserialize<SendProjectStartedNotificationMessage, NotificationSubscriber>(_logger);
+
+        var project = await _dbContext.Projects
+            .Include(projectEntity => projectEntity.CompanyLeadUser)
+            .FirstAsync(p => p.Id == message.ProjectId, ct);
+
+        var companyLeadUser = project.CompanyLeadUser;
+        if (companyLeadUser is not null && companyLeadUser.AquiferNotificationsEnabled)
+        {
+            var templatedEmail = new TemplatedEmail(
+                From: NotificationsHelper.NotificationSenderEmailAddress,
+                // Template Designer: https://mc.sendgrid.com/dynamic-templates/d-7760ec3b5ce34b179384d4783cc1bd81/version/e83075a3-ba61-4d42-922f-9fd5df4ee45c/editor
+                TemplateId: "d-7760ec3b5ce34b179384d4783cc1bd81",
+                DynamicTemplateData: new Dictionary<string, object>
+                {
+                    [EmailService.DynamicTemplateDataSubjectPropertyName] = "Aquifer Notifications: Project Started",
+                    ["aquiferAdminBaseUri"] = _configurationOptions.Value.AquiferAdminBaseUri,
+                    ["projectId"] = project.Id,
+                    ["projectName"] = project.Name,
+                },
+                Tos: [NotificationsHelper.NotificationToEmailAddress],
+                Bccs: [NotificationsHelper.GetEmailAddress(companyLeadUser)]);
+
+            await _emailService.SendEmailAsync(templatedEmail, ct);
+
+            _logger.LogInformation(
+                "Project started notification sent for Project ID {ProjectId} to User ID {UserId}.",
+                project.Id,
+                companyLeadUser.Id);
+        }
+        else
+        {
+            _logger.LogWarning(
```

Comment:
This should _not_ happen. I'd make this an error so we're getting alerted to it right away.